apps/cli/src/commands/send/generic.ts

```diff
@@ -113,6 +114,19 @@ export const registerGenericCommand = (program: Command) => {
                 "input encoding",
             ).choices(["hex", "string", "abi"]),
         )
+        .addOption(
+            new Option(
+                "--type <type>",
+                "Transaction type",
+            ).choices(["evm", "eip712"])
+            .default("evm"),
+        )
+        .addOption(
+            new Option(
+                "--node-base-url <url>",
+                "Node base url",
+            ).default("http://localhost:8080"),
```

Comment:
Shouldn't this option refer to the URL of the transaction submission service to be used?
Maybe something like `--eip712-tx-url`, with a default value of http://localhost:8080/transaction?